app/views/search/bento.html.erb

```diff
@@ -36,7 +36,5 @@
 
   <%= render partial: "trigger_search", locals: { target: 'books', id: 'books_content'} %>
 
-  <%= render partial: "trigger_search", locals: { target: 'whatnot', id: 'whatnot_content'} %>
```

Comment:
We dropped the whatnot profile (recently) and box (a while ago) so i think this was just clean up.